### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#306] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322031-b2ceb35d-3467-44cf-9c22-f4590dc1cac9.png)

**Comment by Exyss** [comment]
**Chiarimento dell'ultimo Step**

Poiché abbiamo 3 * N * (N+1), le casistiche sono due:
1. Se N è un numero pari, allora abbiamo 3 * Pari * Dispari
2. Se N è un numero dispari, allora abbiamo 3 * Dispari * Pari

Poiché ogni numero pari può essere riscritto come 2 * Dispari, in entrambi i casi otteniamo **3 * 2 * Dispari * Dispari**

Dunque in entrambi i casi otteniamo 6 * X, che è divisibile per 6

![Esercizio 6](https://user-images.githubusercontent.com/65043207/152645326-1e9bdabf-c656-447a-926e-15a159303944.jpg)
